src/commands/app/deploy.js

```diff
@@ -83,7 +83,7 @@ class Deploy extends BuildCommand {
       }
 
       // 2. Bail if workspace is production and application status is PUBLISHED, honor force-deploy
-      if (!isStandaloneApp && aioConfig?.project?.workspace?.name === 'Production' && !flags['force-deploy']) {
+      if (libConsoleCLI && !isStandaloneApp && aioConfig?.project?.workspace?.name === 'Production' && !flags['force-deploy']) {
```

Comment:
`libConsoleCLI` check is not needed because of this fix: https://github.com/adobe/aio-cli-plugin-app/commit/3986d3672c5c8d8b87d2bef2ba35b6a068f1c046